pkg/cmd/adm/unregister_member_test.go

```diff
@@ -164,9 +164,13 @@ func TestUnregisterMemberLacksPermissions(t *testing.T) {
 	AssertToolchainClusterSpec(t, fakeClient, toolchainCluster)
 }
 
-func mockRestart(ctx *clicontext.CommandContext, clusterName string) error {
+func mockRestart(ctx *clicontext.CommandContext, clusterName string, cfc ConfigFlagsAndClientGetterFunc) error {
 	if clusterName == "host" && ctx != nil {
 		return nil
 	}
+	_, _, err := cfc(ctx, clusterName)
+	if err == nil {
+		return nil
+	}
```

Comment:
this part is never being called if I'm not missing something

Reply:
updated the logic